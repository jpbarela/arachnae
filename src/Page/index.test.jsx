// @flow
import * as React from "react";
import { render } from "@testing-library/react";
import { Page } from "./index";

describe("Row", () => {
  it("renders", () => {
    const container = render(<Page>I'm a page</Page>);
    expect(container.baseElement.firstChild).toMatchSnapshot();
  });
});
