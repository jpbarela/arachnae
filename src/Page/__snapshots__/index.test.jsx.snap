// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders 1`] = `
<div>
  <div
    class="page_wxvzjb"
  >
    I'm a page
  </div>
</div>
`;
